webviz_config/_oauth2.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, app: flask.app.Flask):
         self._tenant_id = os.environ["WEBVIZ_TENANT_ID"]
         self._client_id = os.environ["WEBVIZ_CLIENT_ID"]
         self._client_secret = os.environ["WEBVIZ_CLIENT_SECRET"]
-        self._scope = os.environ["WEBVIZ_SCOPE"]
+        scope_raw = os.environ["WEBVIZ_SCOPE"]
+        self._scope = [scope.strip() for scope in scope_raw.split(",")]
```

Comment:
```suggestion
        self._scope = [scope.strip() for scope in os.environ["WEBVIZ_SCOPE"].split(",")]
```
Maybe cleaner (saves a local variable + naming is always difficult 🙂)? 

Other than that - LGTM! 👏 Thanks.

Add a changelog entry?